lib/bandit/pipeline.ex

```diff
@@ -216,16 +226,17 @@ defmodule Bandit.Pipeline do
     {:error, error}
   end
 
-  defp handle_error(error, stacktrace, transport, span, opts) do
-    Bandit.Telemetry.span_exception(span, :exit, error, stacktrace)
-    status = error |> Plug.Exception.status() |> Plug.Conn.Status.code()
+  defp handle_error(kind, reason, stacktrace, transport, span, opts) do
+    # TODO: Fix this telementry kind
+    Bandit.Telemetry.span_exception(span, :exit, reason, stacktrace)
+    status = reason |> Plug.Exception.status() |> Plug.Conn.Status.code()
 
     if status in Keyword.get(opts.http, :log_exceptions_with_status_codes, 500..599) do
```

Comment:
Ideally this conditional would be inside the `Bandit.LoggerTranslator`. Need a way to access the `opts.http` from there.